PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/wizards/functions/module/FunctionTriggerChooserStep.java

```diff
@@ -0,0 +1,132 @@
+ * SOFTWARE.
+ *
+ */
+package com.microsoft.intellij.wizards.functions.module;
+
+import com.intellij.ide.util.projectWizard.ModuleWizardStep;
+import com.intellij.ide.util.projectWizard.WizardContext;
+import com.intellij.openapi.options.ConfigurationException;
+import com.intellij.ui.components.JBLabel;
+import com.intellij.ui.table.JBTable;
+import com.intellij.util.ui.FormBuilder;
+import com.intellij.util.ui.JBUI;
+import com.intellij.util.ui.components.BorderLayoutPanel;
+import com.microsoft.intellij.wizards.functions.AzureFunctionsConstants;
+
+import org.apache.commons.lang3.BooleanUtils;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.JComponent;
+import javax.swing.JPanel;
+import javax.swing.JTable;
+import javax.swing.table.DefaultTableModel;
+import javax.swing.table.TableColumn;
+
+import java.awt.BorderLayout;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FunctionTriggerChooserStep extends ModuleWizardStep {
+    private final WizardContext wizardContext;
+    private JTable table;
+
+
+    FunctionTriggerChooserStep(final WizardContext wizardContext) {
+        this.wizardContext = wizardContext;
+    }
+
+    @Override
+    public JComponent getComponent() {
+        final FormBuilder builder = new FormBuilder();
+        builder.addComponent(new JBLabel("Choose Functions Triggers:"));
+
+        table = new JBTable();
+        final DefaultTableModel model = new DefaultTableModel() {
+            final Class<?>[] columnClass = new Class[] { Boolean.class, String.class };
+
+            @Override
+            public boolean isCellEditable(final int row, final int col) {
+                return col == 0;
+            }
+
+            @Override
+            public Class<?> getColumnClass(final int columnIndex) {
+                return columnClass[columnIndex];
+            }
+        };
+        model.addColumn("Selected");
+        model.addColumn("Trigger name");
+        table.setModel(model);
+        setupFunctionTriggers();
+
+        final BorderLayoutPanel customPanel = JBUI.Panels.simplePanel(10, 0);
+        customPanel.addToTop(table);
+        builder.addComponent(customPanel);
+
+        final TableColumn column = table.getColumnModel().getColumn(0);
+        column.setHeaderValue(""); // Don't show title text
+        column.setMinWidth(23);
+        column.setMaxWidth(23);
+        table.getTableHeader().setReorderingAllowed(false);
+
+        final JPanel panel = new JPanel(new BorderLayout());
+        panel.add(builder.getPanel(), "North");
+        return panel;
+    }
+
+    private void setupFunctionTriggers() {
+        final DefaultTableModel model = (DefaultTableModel) table.getModel();
+        boolean isFirst = true;
+        for (final String s : Arrays.asList("HttpTrigger", "BlobTrigger", "QueueTrigger", "TimerTrigger", "EventHubTrigger")) {
+            model.addRow(new Object[] { isFirst, s });
```

Comment:
I think we could change this line to `model.addRow(new Object[] { StringUtils.equals("HttpTrigger",s), s });`, to show HttpTrigger is the default trigger/